examples/server.rs

```diff
@@ -0,0 +1,158 @@
+                tvf_req.put_string(1, req.method().to_string());
+                tvf_req.put_string(2, "/test");
+                HyperResp::SrvReq(String::from("SRV_TEST"), tvf_req)
+            }
+            _ => HyperResp::HttpResp(
+                Response::builder()
+                    .status(404)
+                    .body(BoxBody::new(Full::new(Bytes::from("Not Found"))))
+                    .unwrap(),
+            ),
+        }
+    }
+
+    fn process_srv_response(&self, resp: &M) -> Response<BoxBody<Bytes, Infallible>> {
+        let body = resp
+            .get_string(10)
+            .unwrap_or(Cow::Owned(String::from("empty body")));
+        Response::builder()
+            .body(BoxBody::new(Full::new(Bytes::from(format!(
+                "Body: {body}\nTvfResp: {resp:?}"
+            )))))
+            .unwrap()
+    }
+}
+
+#[settings]
+#[derive(Debug, Default, Deserialize, Serialize, Clone)]
+pub(crate) struct MainHyperSettings {
+    pub(crate) hyper: HyperServerSettings,
+}
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let matches = Command::new("hyper")
+        .version(env!("CARGO_PKG_VERSION"))
+        .author(env!("CARGO_PKG_AUTHORS"))
+        .about(env!("CARGO_PKG_DESCRIPTION"))
+        .arg(arg!(-v - -verbose).action(ArgAction::SetTrue))
+        .arg(arg!(-s --stub "Start a Stub processor").action(ArgAction::SetTrue))
+        .arg(
+            arg!(-c --config <CONFIG_PATH> "Path of the JPTR ProSA configuration file")
+                .default_value("examples/server.yml"),
+        )
+        .get_matches();
+
+    // load the configuration
+    let config = Config::builder()
+        .add_source(config::File::with_name(
+            matches.get_one::<String>("config").unwrap().as_str(),
+        ))
+        .add_source(
+            config::Environment::with_prefix("PROSA")
+                .try_parsing(true)
+                .separator("_")
+                .list_separator(" "),
+        )
+        .build()
+        .unwrap();
+
+    let prosa_hyper_settings = config.try_deserialize::<MainHyperSettings>()?;
+
+    println!("ProSA HYPER settings: {prosa_hyper_settings:?}");
+
+    let filter = TelemetryFilter::default();
+    prosa_hyper_settings.observability.tracing_init(&filter)?;
+
+    // Create bus and main processor
+    let (bus, main) = MainProc::<SimpleStringTvf>::create(&prosa_hyper_settings);
+
+    // Launch the main task
+    debug!("Launch the main task");
+    let main_task = main.run();
+
+    debug!("Start the Hyper processor");
+    let http_proc =
+        HyperServerProc::<SimpleStringTvf>::create(1, bus.clone(), prosa_hyper_settings.hyper);
+    Proc::<HyperDemoAdaptor>::run(http_proc, String::from("hyper"));
+
+    if matches.contains_id("stub") && matches.get_flag("stub") {
+        debug!("Start a Stub processor");
+        // TODO add the SRV_TEST service
+        let stub_proc = StubProc::<SimpleStringTvf>::create_raw(2, bus.clone());
```

Comment:
Missing settings for SRV_TEST service

```suggestion
        let stub_settings = StubSettings::new(vec![String::from("SRV_TEST")]);
        let stub_proc = StubProc::<SimpleStringTvf>::create_raw(2, bus.clone(), stub_settings);
```

Reply:
Implemented the TODO. Now the stub launch a SRV_TEST service from a configuration